### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by azizunnabi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,74 @@
-# Hello-microverse
\ No newline at end of file
+# Hello-microverse
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
```
Comment (mwanawabangona) [inline README.md:5]: - [ ] `required`
- [ ] please add the project description

File: README.md [hunk 2]
```diff
@@ -1 +1,74 @@
...
+> Description the project.
+
+
+## Built With
+
+- Major languages
+- Frameworks
+- Technologies used
```
Comment (mwanawabangona) [inline README.md:13]: - [ ] `required`
- [ ] Here you add the technologies you used to create this project e.g HTML, CSS. kindly make the changes

**Review by mwanawabangona — COMMENTED** [review]
## Status: Changes Required

 ## Hello there @azizunnabi  👋🏽

###  You have done an amazing job so far! 👏🏽 👏🏽


There are some issues that you still need to work on to go to the next project but you are almost there! :pinching_hand:


## Highlights
* ✅  No linter errors :heavy_check_mark:
## Required Changes
 _**Check the comments under the review.**_

- [ ] `required`
- [ ] Hi :wave: its really important that you make descriptive `pull request` please go through this [article](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md) for further understanding. Then I suggest you make a descriptive

- [ ] `required`
- [ ] Kindly make adjustment as requested. 
- Your `readmfile` description is an extremely important aspect of your project. A well-crafted description allows you to show off your work to other developers as well as potential employers.
The quality of a README description often differentiates a good project from a bad project.
see [template](https://github.com/microverseinc/readme-template) 

 ## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### Cheers and Happy coding!
Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @mwanawabangona in your question so I can receive the notification.
>
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
>
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (azizunnabi) [reply README.md:13]: Thanks Sir, I have done what your kindself was suggested

**Pull request merged**